hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -277,10 +277,12 @@ class cip_base_test #(type CFG_T = cip_base_env_cfg,
 
 ### cip_tl_seq_item
 This is extended class of tl_seq_item to generate correct integrity values in
-a_user and d_user. If DUT relies on the agent to generate integrity for TLUL, set
-`cfg.en_tl_intg_gen = 1`, cip_tl_seq_item will override tl_seq_item, and integrity
-values will be generated. If TLUL integrity is handled in the DUT, set
-`cfg.en_tl_intg_gen = 0`, then `a_user` and `d_user` will be fully randomized.
+a_user and d_user. If DUT relies on the agent to generate integrity for TLUL,
+set `cfg.tl_intg_gen_type = TlIntgGenBoth`, cip_tl_seq_item will override
+tl_seq_item, and both integrity values will be generated.
+If TLUL command integrity is handled in the DUT, set
+`cfg.tl_intg_gen_type = TlIntgGenData`, then only data integrity will be
+generated and command integrity will be randomized.
```

Comment:
If the DUT handled the cmd intg computation, but we also computed it anyway, would that cause any harm? I wonder if we can simplify this further, if cmd intg randomization is not strictly needed.

Reply:
An alternate approach would be to "override" the cmd intg bits in the TL driver instead, given that it is a special case for the chip level.

Reply:
If we provide correct cmd intg anyway, we may miss the case that design doesn't compute the cmd intg.

If you prefer a simple quick way, we can just generate both intg and force the cmd intg to random value in tb.sv. 
The current approach is more clear that what is generated in seq and scb also has the info. Either approach is fine to me. WDYT?

Reply:
@sriyerg as discussed, I changed to force cmd intg to random value in tb.sv. Now it's much simpler.